docs/topics/native/native-ios-integration.md

```diff
@@ -1,8 +1,9 @@
-[//]: # (title: iOS integration)
+[//]: # (title: Integration with Swift/Objective-C ARC)
 
-Integration of Kotlin/Native garbage collector with Swift/Objective-C ARC is seamless and generally requires no additional
-work to be done. Learn more about [Swift/Objective-C interoperability](native-objc-interop.md).
+Kotlin and Objective-C have two different memory models, Kotlin uses a garbage collector,
+while Objective-C relies on automatic reference counting (ARC).
```

Comment:
```suggestion
Kotlin and Objective-C use different memory management strategies, Kotlin uses a tracing garbage collector,
while Objective-C relies on automatic reference counting (ARC).
```
How about this?

Don't talk about whole memory models, just about memory management strategies. And also specify that Kotlin uses a *tracing* garbage collector (because ARC is also a form of garbage collection, but without tracing)